source/configuration_manual/event_filter.rst

```diff
@@ -147,6 +151,9 @@ filtering languge.  For example::
   log_debug = (event=http_request_finished AND category=imap) OR \
               (event=imap_command_finished AND user=testuser)
 
+Old global filter syntax
```

Comment:
I'm not sure I like using "old" - since that turns into the "old old old obsolete old previous" kind of mess very quickly. It's probably ok here.

Reply:
what would be alternative?

Reply:
"Obsolete global filter syntax"?  Meh.  That makes it sound a bit like like it's still around, but that's the best adjective I can think of.